app/dashboard/views.py

```diff
@@ -519,15 +519,25 @@ def onboard(request, flow):
     return TemplateResponse(request, 'ftux/onboard.html', params)
 
 
-def dashboard(request):
+def dashboard(request, hackathon=''):
     """Handle displaying the dashboard."""
 
-    keyword = request.GET.get('keywords', False)
-    title = keyword.title() + str(_(" Bounties ")) if keyword else str(_('Issue Explorer'))
+    evt = None
+    if hackathon:
+        try:
+            evt = HackathonEvent.objects.get(name__iexact=hackathon)
+            title = evt.name
+        except:
```

Comment:
Can you switch `except` to be more explicit and catch on: `HackathonEvent.DoesNotExist` and `HackathonEvent.MultipleObjectsReturned` ?  MultipleObjects could probably default to the `last()` obj.